docs/source/api.rst

```diff
@@ -147,6 +147,12 @@ PineconeNeo4jRetriever
 .. autoclass:: neo4j_graphrag.retrievers.external.pinecone.pinecone.PineconeNeo4jRetriever
     :members: search
 
+QdrantNeo4jRetriever
+======================
```

Comment:
[Best practice](https://www.pgdp.net/wiki/RST:_Best_Practices#:~:text=example%20is%20optional.-,Chapters%20and%20Sections,-Chapters%20should%20be) for this to have the same length as the header

```suggestion
====================
```